### DISCUSSION TITLE
Projects (beta): global search doesn't aware of the beta projects

### DISCUSSION THREAD

**Discussion by Totktonada** [community/community#5856] [Projects and Issues]
We triage new issues using the https://github.com/search webpage. The main filters are the following:

* `is:open is:issue` — we're not interested in closed issue here.
* `no:project -milestone:wishlist` — if an issue is in a project or in the wishlist milestone, it is already triaged.
* A lot of `repo:<repository>` filters.

The problem is that issues added to a new project are not excluded by the `no:project` filter. For us any way to exclude issues that are already in a project (a new project) would be okay. It looks as the main blocker for us to move from old projects.

Also I see that `project:<org>/<project_id_from_webpage_url>` does not work with new projects.

There is another way to solve the problem for us, I described it here: https://github.com/github/feedback/discussions/5528

**Comment by Totktonada** [comment]
It seems, now it is tracked in the https://github.com/github/roadmap/issues/287 roadmap issue:

> - [ ] Issue page filters

It is in the 'Future' milestone ATM.

**Reply by Totktonada** [reply]
Or it is tracked here now? https://github.com/github/roadmap/issues/308

Update 2021-11-13: seems no, this issue was deleted.

**Comment by gflarity** [comment]
I opened a similar issue.
